pkg/transaction/transaction.go

```diff
@@ -152,6 +155,19 @@ func (t *transactionService) waitForAllPendingTx() error {
 
 	return nil
 }
+func (t *transactionService) setAccumulativeFee(gasTipCap, gasFeeCap *big.Int, gasLimit uint64) {
+	gasTip := big.NewInt(0)
+	gasFee := big.NewInt(0)
+	if gasTipCap != nil {
+		*gasTip = *gasTipCap
+	}
+	*gasFee = *gasFeeCap
+	t.AccumulativeFee = gasTipCap.Add(gasTip, gasFee.Mul(gasFee, big.NewInt(int64(gasLimit))))
+}
+
+func (t *transactionService) GetAccumulativeFee() *big.Int {
```

Comment:
this returns the fee for ALL transaction which also includes transaction not related to the storage incentives, I don't think this is the right way to do it.